src/main/java/sirius/biz/process/ProcessContext.java

```diff
@@ -217,6 +217,15 @@
      */
     TableOutput addTable(String name, String label, List<Tuple<String, String>> columns);
 
+    /**
+     * Adds a table output to the process.
+     *
+     * @param name    the name of the output
+     * @param label   the label of the output which will be {@link sirius.kernel.nls.NLS#smartGet(String) auto translated}
+     * @return a builder {@link sirius.biz.process.ProcessEnvironment.TableOutputBuilder} to add columns to the table
+     */
+    ProcessEnvironment.TableOutputBuilder addTable(String name, String label);
```

Comment:
shouldn't we move the TableOutputBuilder interface here - as it would be strange that the interface references the impl (?)

Reply:
ah, it's class, then move to top-level?

Reply:
Would move it to TableOutput.Builder

Reply:
Refactored as TableOutput.ColumnBuilder